incubator/azuremonitor-containers/templates/NOTES.txt

```diff
@@ -35,6 +35,7 @@ This deployment will not complete. To proceed, run
 
 {{ .Release.Name }} deployment is complete. 
```

Comment:
We should also specify the cloud here - meaning what cloud did we onboard to, as this is like a confirmation text...